sigs.yml

```
@@ -728,3 +728,92 @@
     gcLiaison:
       - name: 'Austin Parker'
         github: austinlparker
+- name: Localization Teams (part of SIG Communications)
+  sigs:
+  - name: Bengali (bn)
+    meeting: ''
+    notes:
+      type: none
+    chat:
+      - type: slack
+        name: "#otel-localization-bn"
+        id: C08TBCSAY1F
+    sponsors:
+      - name: 'Severin Neumann'
+        github: svrnm
+    gcLiaison:
+      - name: 'Severin Neumann'
+        github: svrnm
+  - name: Chinese (zh-CN)
+    meeting: ''
+    notes:
+      type: none
+    chat:
+      - type: slack
+        name: "#otel-localization-zhcn"
+        id: C08SSK25Y7L
+    sponsors:
+      - name: 'Severin Neumann'
+        github: svrnm
+      - name: 'Reiley Yang'
+        github: reyang
```

```suggestion
      - name: 'Reiley Yang'
        github: reyang
      - name: 'Michael Yao'
        github: windsonsea
```
I would like to join the meeting as a zh-cn approver

---

Yes, I added @reyang here as sponsor although this will not be displayed in the README.md. it's more as an information worth tracking!